src/js/themes/base.js

```diff
@@ -794,6 +794,11 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         // border: undefined,
         // extend: undefined,
       },
+      // row: {
+      //   hover: {
+      //     background: undefined,
```

Comment:
```suggestion
      //     background: undefined,
      //     color: undefined,
```

Reply:
Is there a comment here?

Reply:
yes, I added a suggested code. you can accept it easily via github, or commit it yourself :)